docs/user/reuse_garage_policy.md

```diff
@@ -0,0 +1,163 @@
+
+    runner = LocalRunner(ctxt)
+
+    policy = GaussianMLPPolicy(env.spec,
+                               hidden_sizes=[64, 64],
+                               hidden_nonlinearity=torch.tanh,
+                               output_nonlinearity=None)
+
+    value_function = GaussianMLPValueFunction(env_spec=env.spec,
+                                              hidden_sizes=(32, 32),
+                                              hidden_nonlinearity=torch.tanh,
+                                              output_nonlinearity=None)
+
+    algo = VPG(env_spec=env.spec,
+               policy=policy,
+               value_function=value_function,
+               max_path_length=100,
+               discount=0.99,
+               center_adv=False)
+
+    runner.setup(algo, env)
+    runner.train(n_epochs=100, batch_size=10000)
+    # Save the experiment to continue it later!
+    runner.save(99)
+
+
+vpg_pendulum()
+```
+
+By default, the data is saved to `./data/local/experiment`, but you can
+specify your own directory by setting `snapshot_dir` when instantiating
+the `LocalRunner`.
+
+## Resuming an experiment
+
+Once you've saved your experiment, resuming is easy. The following example was
+pulled from [here](https://github.com/rlworkgroup/garage/blob/175ac4c90a408e2314d91cdbe95e419b183e0684/examples/torch/resume_training.py).
+
+```
+"""This is an example to resume training programmatically."""
+import click
+
+from garage import wrap_experiment
+from garage.experiment import LocalRunner
+
+
+@click.command()
+@click.option('--saved_dir',
+              required=True,
+              help='Path where snapshots are saved.')
+@wrap_experiment
+def resume_experiment(ctxt, saved_dir):
+    """Resume a PyTorch experiment.
+    Args:
+        ctxt (garage.experiment.ExperimentContext): The experiment
+            configuration used by LocalRunner to create the snapshotter.
+        saved_dir (str): Path where snapshots are saved.
+    """
+    runner = LocalRunner(snapshot_config=ctxt)
+    runner.restore(from_dir=saved_dir)
+
+    # You have two options here:
+    # 1. resume immediately using existing parameters
+    runner.resume()
+    # 2. or modify training parameters
+    runner.resume(n_epochs=20)
+
+
+resume_experiment()
+```
+
+## Extracting a trained policy from a snapshot
+
+If instead you wish to extract the trained policy from a saved experiment,
+the process is even simpler. You could do it in three lines like this:
+
+```
+# Load the policy
+import cloudpickle
+data = cloudpickle.load('path/to/snapshot/file')
+policy = data['algo'].policy
+
+# You can also access other components of the experiment
+env = data['env']
+```
+
+Alternatively, you can use a Garage `Snapshotter` instance. It will handle
+the calls to cloudpickle for you, and should continue to work even if we
+change how we pickle (we used to use joblib, for example):
+
+```
+# Load the policy
+from garage.experiment import SnapshotConfig, Snapshotter
```

Comment:
i think you should stick with this one for now, for the exact reason you note (we could change the snapshot format).